source/components/Autocomplete.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import FormField from './FormField';
+import ReactDOM from 'react-dom';
+import events from '../utils/events';
+
+export default class Autocomplete extends FormField {
+
+  static SKIN_PARTS = {
+    INPUT: 'input',
+    AUTOCOMPLETE: 'autocomplete',
+  };
+
+  static propTypes = Object.assign({}, FormField.propTypes, {
+    error: PropTypes.string,
+    maxSelections: PropTypes.number,
+    placeholder: PropTypes.string,
+    suggestedWords: PropTypes.array,
+  });
+
+  state = {
+    selectedWords: [],
+    invalidCharacters: /[^a-zA-Z0-9]/g, // only allow letters and numbers
```

Comment:
I would either move this into a `const INVALID_CHARS` or make it configurable via a prop